content/assignments/minishell/util/parser/DisplayStructure.cpp

```diff
@@ -0,0 +1,186 @@
+// SPDX-License-Identifier: BSD-3-Clause
+
+#include <iostream>
+#include <iomanip>
+#include <cstdio>
+#include <cstdlib>
+#include <string>
+#include <cassert>
+#include "parser.h"
+
+#ifdef UNICODE
+#  error "Unicode not supported in this source file!"
+#endif
+
+using namespace std;
```

Comment:
This is kind of a bad practice. It's recommended that you use `std::cout` etc.

Reply:
Done.